editor/export/editor_export_platform.cpp

```diff
@@ -1636,6 +1736,12 @@ Error EditorExportPlatform::save_pack(const Ref<EditorExportPreset> &p_preset, b
 		return err;
 	}
 
+	if (pd.file_ofs.is_empty()) {
+		DirAccess::remove_file_or_error(tmppath);
+		add_message(EXPORT_MESSAGE_ERROR, TTR("Save PCK"), TTR("No files or changes to export."));
```

Comment:
There is no equivalent error when exporting ZIP, which results in empty archive. Is this intended?

Reply:
I went ahead and added this error to ZIP files as well, which sort of required writing the ZIP file to a temporary file first, much like what the PCK files are doing already, and then moving it when successful.